### PR TITLE
chore(merge): merge from 1.x to master

### PR DESCRIPTION
**Pull request by scottdickerson**
all the fixes in 1.x merged to master

### CODE REVIEW

File: .storybook/Container.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default class Container extends Component {
           padding: '3rem',
           display: 'flex',
           flexDirection: 'column',
-          alignItems: 'center',
+          alignItems: 'stretch',
```
Comment (tay1orjones) [inline .storybook/Container.js]: Does this conflict with the new wrapper component that some stories use to be full width? I think we want the smaller components to still be centered.

File: README.MD [hunk 2]
```diff
@@ -98,7 +98,7 @@ Usage of this package does not require you to install additional Carbon packages
 
 ## 📚 Docs
 
-You can find more information about how to use each component by checking out our [Storybook](https://IBM.github.io/carbon-addons-iot-react/).
+You can find more information about how to use each component by checking out our [Storybook](https://1-x.carbon-addons-iot-react.com/).
```
Comment (tay1orjones) [inline README.MD]: Incorrect URL, should be https://carbon-addons-iot-react.com

File: src/components/Hero/Hero.jsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  rightContentBreadcrumb,
+  tooltip,
+  isLoading,
+  i18n: { closeLabel },
+  onClose,
+}) => {
+  if (__DEV__) {
+    warning(
+      false,
+      'PageHero component has been deprecated and will be removed in the next release of `carbon-addons-iot-react`. \n Refactor to use PageTitleBar component instead.'
```
Comment (tay1orjones) [inline src/components/Hero/Hero.jsx]: Should be `Hero`, not `PageHero`.

File: src/components/Hero/Hero.story.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+const breadcrumb = [<a href="/">Home</a>, <a href="/">Type</a>, <span>Instance</span>];
+
+const tooltip = {
+  message:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ut auctor tortor, et condimentum dolor.',
+  href: '/',
+  linkLabel: 'Learn more',
+};
+
+storiesOf('Watson IoT|Hero', module)
```
Comment (tay1orjones) [inline src/components/Hero/Hero.story.jsx]: Needs `deprecated` in the title
`Watson IoT|Hero (Deprecated)`

File: src/components/Dashboard/Dashboard.story.jsx [hunk 5]
```diff
@@ -618,6 +689,42 @@ storiesOf('Watson IoT Experimental|Dashboard', module)
       </FullWidthWrapper>
     );
   })
+  .add('full screen table card', () => {
```
Comment (tay1orjones) [inline src/components/Dashboard/Dashboard.story.jsx]: When the table is scrolled, the pagination sticks and overlays rows
image

File: src/components/Page/EditPage.story.jsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { text } from '@storybook/addon-knobs';
+
+import EditPage from './EditPage';
+
+const commonEditPageProps = {
+  title: 'Page Title',
+  onClose: action('onClose'),
```
Comment (tay1orjones) [inline src/components/Page/EditPage.story.jsx]: When clicked, this console error occurs
```
Uncaught TypeError: toISOString is not a function
    at String.toJSON (<anonymous>)
    at Object.<anonymous> (index.umd.js:1)
    at JSON.stringify (<anonymous>)
    at stringify (index.js:353)
    at PostmsgTransport.send (index.js:148)
    at handler (index.js:126)
    at Channel.emit (index.js:136)
    at actionHandler (action.js:47)
    at Object.ca (react-dom.production.min.js:14)
    at ja (react-dom.production.min.js:15)
```

File: src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx [hunk 7]
```diff
@@ -43,6 +43,21 @@ storiesOf('Watson IoT|TableBodyRow', module)
       options={{ hasRowActions: true, hasRowExpansion: true }}
     />
   ))
+  .add('is not selectable', () => (
+    <TableBodyRow
```
Comment (tay1orjones) [inline src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx]: Checkboxes here aren't aligned properly, though they appear correct in the StatefulTable story both in unselectable and selectable state.

image

File: src/components/Table/Table.story.jsx [hunk 8]
```diff
@@ -362,6 +362,33 @@ storiesOf('Watson IoT|Table', module)
       },
     }
   )
+  .add(
+    'Stateful Example with every third row unselectable',
+    () => (
+      <StatefulTable
```
Comment (tay1orjones) [inline src/components/Table/Table.story.jsx]: alignment of the checkboxes is off, as well as the left border of the row, is that intended?
image

File: src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx (hunk 7 above)
Comment (tay1orjones) [reply src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx]: This is probably related https://github.com/IBM/carbon-addons-iot-react/pull/592/files?file-filters%5B%5D=.jsx#r332114465

File: src/components/Page/EditPage.story.jsx (hunk 6 above)
Comment (tay1orjones) [reply src/components/Page/EditPage.story.jsx]: This isn't an issue with just this component, in fact, all actions are broken for all the stories. I made a new issue, #594

File: src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx (hunk 7 above)
Comment (scottdickerson) [reply src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx]: this ones does actually appear to be broken in the underlying stateful table as well.  I think this is due to the wrong checkbox being used.

File: src/components/Dashboard/Dashboard.story.jsx (hunk 5 above)
Comment (scottdickerson) [reply src/components/Dashboard/Dashboard.story.jsx]: fixed this now

**Pull request merged**